### PR TITLE
rename layouts and add Sorted Run Duration

### PR DESCRIPTION
**Pull request by brandonreid**
https://github.com/PrefectHQ/graphs/assets/6776415/d90d4403-95ea-4282-853f-bc9b31461a89

### CODE REVIEW

File: src/workers/layouts/horizontal.ts [hunk 1]
```diff
@@ -54,3 +58,18 @@ function getHorizontalTimeLayout({ data, horizontalSettings }: ClientLayoutMessa
   return layout
 }
 
+function getHorizontalLeftAlignedLayout({ data, horizontalSettings }: ClientLayoutMessage): HorizontalLayout {
+  const scale = horizontalScaleFactory(horizontalSettings)
+  const layout: HorizontalLayout = new Map()
+
+  const x = scale(data.nodes.entries().next().value[1].start_time ?? new Date())
```
Comment (pleek91) [inline src/workers/layouts/horizontal.ts:65]: This just gets the start time of the first entry right?

**Comment by pleek91** [comment]
@brandonreid This appears to remove the ability to do dependency + waterfall. Is that intentional?

File: src/components/RunGraphSettings.vue [hunk 2]
```diff
@@ -62,43 +54,33 @@
...
-      return layout.horizontal
-    },
-    set(value) {
-      setHorizontalMode(value)
-    },
-  })
+  type LayoutOption = `${HorizontalMode}_${VerticalMode}`
 
-  const verticalOptions: Option<VerticalMode>[] = [
+  const layoutOptions: Option<LayoutOption>[] = [
```
[the remark this answers is not in the document]
Comment (brandonreid) [reply src/components/RunGraphSettings.vue:59]: That's a good call, I'll throw the names out there, maybe a Will or others have better ideas on these.

**Comment by brandonreid** [comment]
> @brandonreid This appears to remove the ability to do dependency + waterfall. Is that intentional?

@pleek91 Yeah, open to pushback there, but the dependency waterfall seemed like an irrelevant way to view things.. I guess if you care about run order?

**Comment by pleek91** [comment]
> > @brandonreid This appears to remove the ability to do dependency + waterfall. Is that intentional?
> 
> @pleek91 Yeah, open to pushback there, but the dependency waterfall seemed like an irrelevant way to view things.. I guess if you care about run order?

I'd personal like to not have waterfall at all. But its the fastest layout which I think is the only reason its there. For performance on large graphs.

File: src/workers/layouts/horizontal.ts [hunk 3]
```diff
@@ -54,3 +58,18 @@ function getHorizontalTimeLayout({ data, horizontalSettings }: ClientLayoutMessa
   return layout
 }
 
+function getHorizontalLeftAlignedLayout({ data, horizontalSettings }: ClientLayoutMessage): HorizontalLayout {
+  const scale = horizontalScaleFactory(horizontalSettings)
+  const layout: HorizontalLayout = new Map()
+
+  const firstEntryStartTime = data.nodes.entries().next().value[1].start_time
```
Comment (pleek91) [inline src/workers/layouts/horizontal.ts:65]: Could this just use the start time of the run itself? `data.start_time`

Comment (brandonreid) [reply src/workers/layouts/horizontal.ts:65]: ugg... totally. haha updating

**Review by pleek91 — APPROVED** [review]

**Pull request merged**